app/assets/javascripts/discourse/app/lib/sidebar/admin-nav-map.js

```diff
@@ -3,12 +3,11 @@ import getURL from "discourse-common/lib/get-url";
 export const ADMIN_NAV_MAP = [
   {
     name: "plugins",
-    route: "adminPlugins.index",
     label: "admin.plugins.title",
     links: [
       {
         name: "admin_installed_plugins",
-        route: "adminPlugins",
```

Comment:
O isn't this the bug here because we used `adminPlugins` instead of `adminPlugins.index`? Now that we change the route to `adminPlugins.index`, do we still need `currentWhen`?

Reply:
Still happens the same either way, I just didn't want this to be ambiguous